src/sbt-test/sbt-scalafix/scalafixEnable/build.sbt

```diff
@@ -15,7 +15,7 @@ lazy val scala211_old = project.settings(
 
 // 2.11.x is supported
 lazy val scala211 = project.settings(
-  // semanticdb-scalac_2.11.11 no longer available, became available as of 2.0.0
```

Comment:
The fact that there the earliest artifact was non binary-compatible exercised more code, so the change of behavior reduces code coverage unfortunately. Extracting the logic and stubbing the coursier API seems like a lot of work though, so I think we can live with that, and revisit if/when scalameta reaches 5.x.